WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -452,8 +465,14 @@ private void startLogin() {
     // LoginPrologueListener implementation methods
 
     @Override
-    public void showEmailLoginScreen() {
-        checkSmartLockPasswordAndStartLogin();
+    public void showEmailLoginScreen(@NonNull Context context) {
+        CustomTabsIntent intent = new CustomTabsIntent.Builder()
+                .setShareState(CustomTabsIntent.SHARE_STATE_OFF)
+                .setStartAnimations(this, R.anim.activity_slide_up_from_bottom, R.anim.activity_slide_up_from_bottom)
```

Comment:
For consistency with the other logic screens, WDYT about changing this to slide in from the right, slide out to the left?